eng/helix/content/runtests.sh

```diff
@@ -25,8 +27,44 @@ export DOTNET_CLI_HOME="$DIR/.home$RANDOM"
 
 export DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1
 
-# Set playwright browser path
+# Set playwright stuff
 export PLAYWRIGHT_BROWSERS_PATH="$DIR/ms-playwright"
+if [[ "$helixQueue" == *"OSX"* ]]; then
+    export PLAYWRIGHT_DRIVER_PATH="$DIR/.playwright/osx/native/playwright.sh"
+    PLAYWRIGHT_NODE_PATH=$DIR/.playwright/osx/native/node
+else
+    export PLAYWRIGHT_DRIVER_PATH="$DIR/.playwright/unix/native/playwright.sh"
+    PLAYWRIGHT_NODE_PATH=$DIR/.playwright/unix/native/node
+fi
+export InstallPlaywright="$installPlaywright"
+if [ -f "$PLAYWRIGHT_DRIVER_PATH" ]; then
```

Comment:
Suggest removing this and undoing one level of nesting because `$PLAYWRIGHT_DRIVER_PATH` cannot be empty

Reply:
I believe the -f check is testing for existing of the file on disk, at least that's what its supposed to be doing :)  Only a playwright test project would have the driver file in the payload

Reply:
Oops, I missed the `-f`